lib/+~<>.rb

```diff
@@ -0,0 +1 @@
+'foo';def a;end;def a;end
```

Comment:
__[staging][rubocop]__
* Lint/Void: Literal `'foo'` used in void context.
* Lint/DuplicateMethods: Method `Object#a` is defined at both /sideci/sideci_ruby_sandbox/lib/+~<>.rb:1 and /sideci/sideci_ruby_sandbox/lib/+~<>.rb:1.

<!-- Comment by [SideCI](https://sideci.com) -->